L1Trigger/L1TTrackerDTC/python/Producer_cff.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+#---------------------------------------------------------------------------------------------------------
+# This describes the DTC Stub processing
+#---------------------------------------------------------------------------------------------------------
+
+#=== Import default values for all parameters & define EDProducer.
+
+from L1Trigger.L1TTrackerDTC.Producer_Defaults_cfi import L1TTrackerDTCProducer_params
+
+L1TTrackerDTCProducer = cms.EDProducer( 'L1TTrackerDTCProducer', L1TTrackerDTCProducer_params )
+
+if L1TTrackerDTCProducer_params.ParamsED.DataFormat == "Hybrid":
```

Comment:
this is an unusual way to modify a producer. What are the use cases? (In this PR, it looks like DataFormat is only ever "Hybrid"...)

Reply:
There are different track finding algorithms. The default is "Hybrid", which should be used during RelVal sample production. Any private user code using "TMTT" has to rerun the DTC step.

Reply:
It seems like this would be better accomplished with a customization function. Right now, the user would have to edit `Producer_Defaults_cfi.py`.

Reply:
I thought that is the purpose of configuration files. Could you please explain in more detail how you would restructure the code? I wanted to separate the two configurations as much as possible while sharing as much code as possible.

Reply:
* change this file so it only has what you want to be the default
* add a separate file called `customizeProducer_cff.py` (or whatever), with a function for each non-default case:
```python
def useHybrid(process):
    from L1Trigger.TrackerDTC.Producer_Defaults_cfi import TrackerDTCProducer_params
    from L1Trigger.TrackerDTC.Format_Hybrid_cfi import TrackerDTCFormat_params
    process.TrackerDTCProducer = cms.EDProducer('TrackerDTCProducer', TrackerDTCProducer_params, TrackerDTCFormat_params )
    return process
```

Then the user can just put `--customise L1Trigger.TrackerDTC.customizeProducer_cff.useHybrid` in their `cmsDriver.py` command, or import the function by hand if using an existing `cmsRun` config.